aten/src/ATen/native/mkldnn/Utils.cpp

```diff
@@ -0,0 +1,36 @@
+#include <ATen/native/mkldnn/Utils.h>
+#include <THNN/generic/pooling_shape.h>
+
+#if AT_MKLDNN_ENABLED()
+
+namespace at { namespace native {
+
+std::vector<int64_t> expand_param_if_needed(IntArrayRef param, int64_t expected_dim) {
+  if (param.size() == 1) {
+    return std::vector<int64_t>(expected_dim, param[0]);
+  } else {
+    return param.vec();
+  }
+}
+
+std::vector<int64_t> conv_output_size(
```

Comment:
why did this move?  This has nothing to do with mkldnn, right?

Reply:
this was in mkldnn/Conv.cpp before and only used in mkldnn_convolution